sdks/java/core/src/test/java/org/apache/beam/sdk/util/FileIOChannelFactoryTest.java

```diff
@@ -136,8 +136,11 @@ public void testMatchNone() throws Exception {
     temporaryFolder.newFile("aa");
     temporaryFolder.newFile("ab");
 
-    // Windows doesn't like resolving paths with * in them, so the * is appended after resolve.
-    assertThat(factory.match(factory.resolve(temporaryFolder.getRoot().getPath(), "b") + "*"),
+    // Windows doesn't like resolving paths with * in them,
```

Comment:
Is this still true for the new implementation? (also in other places in this file)